README.md

```diff
@@ -439,6 +441,9 @@ class Sabberworm\CSS\CSSList\Document#4 (2) {
 
 ```
 
+</details>
+
+
```

Comment:
For consistency, please let's not have two blank lines in a row in the README.

Reply:
The extra newline is gone now :)